src/app/core/app.component.ts

```diff
@@ -49,6 +49,10 @@ export class AppComponent implements OnInit, OnDestroy {
     private renderer: Renderer2,
     private el: ElementRef,
   ) {
+    if (window.name === 'updateProfileWindow') {
+      (window.opener as Window).dispatchEvent(new Event('profileUpdated'));
+    }
+
```

Comment:
Over time it will be difficult to understand why this is here.
We should find a way to make understand where this comes from, like moving it to a current-user helper in the group module for instance.

Reply:
Fixed